hoot-core/src/main/cpp/hoot/core/io/OsmApiReader.h

```diff
@@ -0,0 +1,98 @@
+/*
+ * This file is part of Hootenanny.
+ *
+ * Hootenanny is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * --------------------------------------------------------------------
+ *
+ * The following copyright notices are generated automatically. If you
+ * have a new notice to add, please use the format:
+ * " * @copyright Copyright ..."
+ * This will properly maintain the copyright information. DigitalGlobe
+ * copyrights will be updated automatically.
+ *
+ * @copyright Copyright (C) 2019 DigitalGlobe (http://www.digitalglobe.com/)
+ */
+#ifndef OSM_API_READER_H
+#define OSM_API_READER_H
+
+// Hoot
+#include <hoot/core/elements/Tags.h>
+#include <hoot/core/io/OsmXmlReader.h>
+#include <hoot/core/io/ParallelBoundedApiReader.h>
+#include <hoot/core/util/Units.h>
+
+// Qt
+#include <QHash>
+#include <QString>
+
+namespace hoot
+{
+
+class OsmMap;
+class Element;
+
+/**
+ * Reads OSM XML output from an OSM API map call by bounding box.  Reading will fail without a bounding box
+ * https://wiki.openstreetmap.org/wiki/API_v0.6#Retrieving_map_data_by_bounding_box:_GET_.2Fapi.2F0.6.2Fmap
+ */
+class OsmApiReader : public OsmXmlReader, private ParallelBoundedApiReader
```

Comment:
Using `private` inheritance because none of the methods from `ParallelBoundedApiReader` need to be called outside of the class.  All access can be done through the `OsmMapReader` interface.